tests/test_invoices.py

```diff
@@ -15,7 +15,7 @@ def test_invoice(node_factory, chainparams):
     l1, l2 = node_factory.line_graph(2, fundchannel=False, opts={'log-level': 'io'})
 
     addr1 = l2.rpc.newaddr('bech32')['bech32']
-    addr2 = l2.rpc.newaddr('p2sh-segwit')['p2sh-segwit']
+    addr2 = '2MxqzNANJNAdMjHQq8ZLkwzooxAFiRzXvEz' if not chainparams['elements'] else 'XGx1E2JSTLZLmqYMAo3CGpsco85aS7so33'
```

Comment:
we would like to introduce a top constant for the different chains?

I think we should have already something, I remember when I did the change of the default port, but I did not remember where this constant are

Reply:
It's only two places and it's better to be explicit, IMHO.